secant/Screens/Onboarding/OnboardingScreenViewModel.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  OnboardingScreenViewModel.swift
+//  secant-testnet
+//
+//  Created by Francisco Gindre on 9/17/21.
+//
+
+import Foundation
+import Combine
+
+protocol OnboardingStepProviding {
+    var totalSteps: Int { get }
+    var currentStep: OnboardingStep { get }
+    var hasNext: Bool { get }
+    var hasPrevious: Bool { get }
+    func next()
+    func previous()
+}
+
+struct OnboardingStep {
+    var title: String?
+    var imageName: String
+    var blurb: String
+    var stepNumber: Int
+}
+
+class OnboardingScreenViewModel: BaseViewModel<OnboardingStepProviding>, ObservableObject {
+    enum RightBarButton {
+        case skip
+        case close
+        case none
+    }
+    @Published var currentStep: OnboardingStep
```

Comment:
This should have a space above it because its a new section in the class structure.